sdks/java/io/file-based-io-tests/pom.xml

```diff
@@ -145,6 +145,89 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>io-it-hdfs-small</id>
```

Comment:
Please add instructions for invoking the test manually.

Reply:
ok